src/include/platform/internal/GenericConfigurationManagerImpl.cpp

```diff
@@ -623,6 +706,45 @@ void GenericConfigurationManagerImpl<ConfigClass>::LogDeviceConfig()
         ChipLogProgress(DeviceLayer, "  Setup Discriminator: %" PRIu16 " (0x%" PRIX16 ")", setupDiscriminator, setupDiscriminator);
     }
 
+    {
+        uint32_t iterationCount;
+        if (GetSpake2pIterationCount(iterationCount) != CHIP_NO_ERROR)
+        {
+            iterationCount = 0;
+        }
+        ChipLogProgress(DeviceLayer, "  Spake2p Iteration Count: %" PRIu32 "", iterationCount);
+    }
+
+    {
+        uint8_t salt[chip::kPBKDFMaximumSaltLen];
+        size_t saltLen;
+        err = GetSpake2pSalt(salt, sizeof(salt), saltLen);
+        if (err == CHIP_NO_ERROR)
+        {
+            // TODO: How to log uint8_t array?
+            ChipLogProgress(DeviceLayer, "  Spake2p Salt: %" PRIu64, reinterpret_cast<uint64_t *>(salt)[0]);
+        }
+        else
+        {
+            ChipLogProgress(DeviceLayer, "  Spake2p Salt: (not set)");
+        }
+    }
+
+    {
+        uint8_t verifier[sizeof(PASEVerifier)];
+        size_t verifierLen;
+        err = GetSpake2pVerifier(verifier, sizeof(verifier), verifierLen);
+        if (err == CHIP_NO_ERROR)
+        {
+            // TODO: How to log uint8_t array?
+            ChipLogProgress(DeviceLayer, "  Spake2p Verifier: %" PRIu64, reinterpret_cast<uint64_t *>(verifier)[0]);
+        }
+        else
+        {
+            ChipLogProgress(DeviceLayer, "  Spake2p Verifier: (not set)");
+        }
+    }
```

Comment:
We should not need to log these, as they are not arguments to anything, and would cause more coupling of spake config with the temporary QR code URL logging. Suggest removing this

Reply:
removed